Docs/source/usage/parameters.rst

```diff
@@ -228,6 +228,16 @@ Overall simulation parameters
         \boldsymbol{\nabla}^2 \phi = - \rho/\epsilon_0 \qquad \boldsymbol{E} = - \boldsymbol{\nabla}\phi \\
         \boldsymbol{\nabla}^2 \boldsymbol{A} = - \mu_0 \boldsymbol{j} \qquad \boldsymbol{B} = \boldsymbol{\nabla}\times\boldsymbol{A}
 
+    * ``labframe-semi-implicit``: Poisson's equation is solved with a modified dielectric function
+      to create a semi-implicit scheme in which grid heating effects due to unresolved
```

Comment:
Could you add a reference to the appendix A of Dan Barnes' paper?
Also: maybe we should discuss in more detail what we mean by "unresolved" plasma modes here? Are these modes that are not resolved in space or in time?

Reply:
Also, is there any guidance for how to set `C_{SI}`? And could you define "marginally stable"?

Reply:
Also: I wonder if there is a risk of confusion with `algo.evolve_scheme = semi_implicit_em`, which is not directly related.

Reply:
Could you also include how the documentation for how to set the C_SI from the input script?